validation/cross_validate_rust_python.py

```
@@ -146,8 +176,25 @@ def giersiepen_python(shear_stress, exposure_time):
+
+    for tau, t in test_cases:
+        damage_python = giersiepen_python(tau, t)
+        damage_rust = hemolysis.damage_index(tau, t)
+        diff_pct = abs(damage_python - damage_rust) / max(damage_python, 1e-10) * 100
+
+        print(f"{tau:12.1f} {t:12.2f} {damage_rust:15.6f} {diff_pct:12.6f}%")
+        if diff_pct >= 1e-4:
+            passed = False
+
+    if passed:
+        print(f"  ✓ Cross-validation passed")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)